fighters/dedede/src/acmd/ground.rs

```diff
@@ -1,97 +1,97 @@
 
 use super::*;
 
-unsafe extern "C" fn dedede_attack_11_game(fighter: &mut L2CAgentBase) {
```

Comment:
`dedede_attack_11_game` needs to change to `game_attack11`

Similar story for all other acmds